### PR TITLE
Eliran/raw number fixes

### PR DESCRIPTION
**Pull request by thehobbit85**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/components/scenes/CryptoExchangeScene.js [hunk 1]
```diff
@@ -295,69 +295,80 @@ const DIVIDE_PRECISION = 18
...
-      toFiatToCrypto = getExchangeRate(state, toPrimaryInfo.exchangeCurrencyCode, toWallet.isoFiatCurrencyCode)
-    } else {
-      toCurrencyCode = ''
-      toExchangeAmount = ''
-      toPrimaryInfo = emptyCurrencyInfo
-      toButtonText = s.strings.select_recv_wallet
-      toFiatToCrypto = 1
+      const { toNativeAmount, toWalletPrimaryInfo } = cryptoExchange
+      const {
+        displayDenomination: { name },
```
Comment (peachbits) [inline src/components/scenes/CryptoExchangeScene.js:338]: fix names

File: src/components/scenes/CryptoExchangeScene.js [hunk 2]
```diff
@@ -295,69 +295,80 @@ const DIVIDE_PRECISION = 18
...
-      fromPrimaryInfo = state.cryptoExchange.fromWalletPrimaryInfo
-      fromNativeAmount = state.cryptoExchange.fromNativeAmount
-      fromButtonText = fromWallet.name + ':' + fromCurrencyCode
-      fromExchangeAmount = bns.div(fromNativeAmount, fromPrimaryInfo.exchangeDenomination.multiplier, DIVIDE_PRECISION)
-      fromFiatToCrypto = getExchangeRate(state, fromPrimaryInfo.exchangeCurrencyCode, fromWallet.isoFiatCurrencyCode)
-
-      const currencyCode = state.cryptoExchange?.fromWallet?.currencyCode
+      const { fromNativeAmount: nativeAmount, fromWalletPrimaryInfo } = cryptoExchange
+      const {
+        displayDenomination: { name },
```
Comment (peachbits) [inline src/components/scenes/CryptoExchangeScene.js:310]: fix names

File: src/components/themed/WalletListCurrencyRow.js [hunk 3]
```diff
@@ -71,7 +71,7 @@ export const getRate = (getRateParams: GetRatesParams) => {
   // Return the Exchange Rate without `percentageString` in case we are missing yesterday's rates
   if (yesterdayUsdExchangeRate === 0 || fiatExchangeRate === 0) return result()
```
Comment (peachbits) [inline src/components/themed/WalletListCurrencyRow.js:72]: change to strings

**Review by peachbits — CHANGES_REQUESTED** [review]

File: src/components/themed/WalletListCurrencyRow.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 // @flow
```
Comment (samholmes) [inline src/components/themed/WalletListCurrencyRow.js:1]: Change commit message to say "Fix WalletListCurrencyRow.js functions to use bns operations"

**Review by samholmes — APPROVED** [review]

**Pull request merged**